netci.groovy

```diff
@@ -2165,7 +2165,7 @@ combinedScenarios.each { scenario ->
                     if (testBuildScenario == 'coverage' || testBuildScenario == 'pri1r2r'|| testBuildScenario == 'gcstress15_pri1r2r') {
                         testBuildScenario = 'pri1'
                     }
-                    else if ( testBuildScenario == 'r2r' || isLongGc(testBuildScenario)) {
+                    else if ( testBuildScenario == 'r2r' || Constants.jitStressModeScenarios.containsKey(testBuildScenario) || isLongGc(testBuildScenario)) {
```

Comment:
The r2r stress modes are in a different list.  This should be: Constants.r2rJitStressScenarios.indexOf(testBuildScenario) != -1